### ISSUE TITLE
Accessibility: [MAS2.1.1]: User is not able to navigate inside the 'Shortcuts' list with keyboard.

### ISSUE THREAD

**Issue by tylerbutler** [microsoft/FluidFramework#3817]
**Build details:**

**OS Version:** 2004(20221.1000)

**Edge Dev Version**: 87.0.654.0 (Official build) dev (64-bit)

**URL:** <https://fluidframework.com/playground/?path=/docs/welcome--page>
  

**Repro Steps:**

1\. Navigate to 'Shortcuts' button in left nav and invoke it.

2.Now try to navigate inside the list using down arrow key and observe.

  
**Actual Result:**  

User is not able to navigate inside the 'Shortcuts' list with keyboard. The list is getting collapsed when users presses down arrow key or tab.  

  

**Expected Result:**  

User should be able to navigate inside the 'Shortcuts' list with keyboard.  

**User Impact:**  

Users with motor impairment who are dependent on keyboard will not able to access the options inside the list if they are not navigable with keyboard.

  

**MAS Reference:**

**MAS 2.1.1 - Keyboard**

**Comment by micahgodbolt** [comment]
Ahh, these shortcuts:

![image](https://user-images.githubusercontent.com/1434956/95490955-d542f900-094d-11eb-81d7-8f016be0a924.png)

**Comment by micahgodbolt** [comment]
this functionality is 100% within Storybook. Should we move this issue into the storybook issue queue?

**Comment by tylerbutler** [comment]
No, we're tracking all issues in this repo centrally. We'll just open PRs against the Storybook repo to close these issues.

**Comment by micahgodbolt** [comment]
Ok, thats fine. Just making the note here that it's an upstream fix needed to resolve this ticket

**Comment by micahgodbolt** [comment]
TBH, I don't know if Storybook would consider this a bug. It's more a clickable tooltip than a menu. I'm not sure how actionable it'll be here in our issue queue, but at least we're tracking it.

**Comment by micahgodbolt** [comment]
Issue logged on Storybook issue queue https://github.com/storybookjs/storybook/issues/12819

**Comment by tooppaaa** [comment]
In progress !

**Comment by micahgodbolt** [comment]
External issue is getting some traction. Hopefully this will mean a fix coming soon.

**Comment by tooppaaa** [comment]
https://github.com/storybookjs/storybook/pull/12904 If you want to help on the review you're welcome ! 
I still have another round of checks (side effects, using a11y tools...)

**Comment by micahgodbolt** [comment]
One more item to note is that the button itself needs to inform the user that it envokes a menu. In Flunet UI we currently do that via `aria-expanded="false" aria-haspopup="true"`

This should probably be added in this PR.
